packages/wallets/src/features/cfd/screens/MT5TradeScreen/MT5TradeScreen.scss

```diff
@@ -33,13 +33,15 @@
 
             &__icon {
                 & > svg {
-                    transform: scale(0.7) translateX(-0.7rem);
+                    width: 2.4rem;
+                    height: 2.4rem;
```

Comment:
Can you check if it's using quill-icon. Rather than controlling it from here you can just pass the props.

Reply:
These icons are taken from the PlatformDetails and MarketTypeDetails constants mapper, which are used in various other components with the size lg (48x48). Passing props would require a reformat of the constants mapper only for usage in this component which is why the size for this icon is only controlled separately from this component